plugins/postgres/dbt/adapters/postgres/impl.py

```diff
@@ -16,6 +16,7 @@
 @dataclass
 class PostgresConfig(AdapterConfig):
     unlogged: Optional[bool] = None
+    indexes: Optional[List[Dict[str, Any]]] = None
```

Comment:
That makes sense. I'll go with leaving it as is and not doing any validation since it will mean less to keep in sync between dbt code and postgres docs and newer versions. The error shown for an incorrect type will be thrown by postgres and should make sense to the user.